### PR TITLE
Feat: Starting 2024 information basics

### PR DESCRIPTION
**Pull request by cwright248**
Commented out 2023 information and changed to 2024 theme. 

Added a sentence in the first section thanking those who attended this year's conference to lead into the link for the 2023 recordings.

### CODE REVIEW

File: content/_index.html [hunk 1]
```diff
@@ -294,27 +294,28 @@ <h1>Irresistible Influence</h1>
...
 	<p>Each year GYC West hosts a young adult retreat to gain strength in the faith, network safely with like-minded believers, and seek a closer communion with our Father.
-		For 2023 the theme is "Irresistible Influence."
-		More details about this event and the theme will come soon.</p>
-	<p>Join us in 2023 to come away from your work and routines and seek revival together.</p>
+		For 2024 the theme will be "Turning Hearts."
+		More details about this event and the theme to come.</p>
+	<p>Join us in 2024 to come away from your work and routines and seek revival together.</p>
 </section>
-<section id="speakers">
+<!--<section id="speakers">
```
Comment (kf6kjg) [inline content/_index.html]: SGML comments require a space between the dashes and any content.

Comment (cwright248) [reply content/_index.html]: Oops, missed that one.

Comment (cwright248) [reply content/_index.html]: Ok, I fixed it.

**Review by kf6kjg — COMMENTED** [review]
I think there's supposed to be some changes in other files. Check the `config.toml` file and look around at the other files.

IIRC every time I started a new year I also started an archive section for the old year so that it doesn't get lost.

Also don't forget to update the header block on every html content file you change. Typically all you need to so it update the date, but in this case you also need to update the title.

File: content/_index.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 +++
-title = "Irresistible Influence, 2023"
+title = "Turning Hearts, 2024"
 description = "GYC West is a regional GYC conference in the Western part of the United States."
-date = "2022-12-21"
+date = "2022-08-30"
```
Comment (kf6kjg) [inline content/_index.html]: 2022?

File: content/archives.html [hunk 3]
```diff
@@ -10,6 +10,34 @@
...
+				<li>Cheyna Ashe</li>
+				<li>Aron Crews</li>
+				<li>K'dee Crews</li>
+				<li>Stephen Gutierrez</li>
+				<li>Anil Kanda</li>
+				<li>Hannah and Jerry Rasmussen</li>
+			</ul>
+		</p>
+		<p><b>Location:</b>Albion Field Station, CA.</p>
+		<p><b>Text:</b>"And I, if I am lifted up from the earth, will draw all peoples to Myself." — John 12:32</p>
```
Comment (kf6kjg) [inline content/archives.html]: Missing space after the closing bold tags, causes the text to jam together,

**Review by kf6kjg — CHANGES_REQUESTED** [review]
Very minor changes, over-all looking good - you even caught stuff I'd not mentioned! :D

**Review by kf6kjg — APPROVED** [review]

**Pull request merged**